linux_os/guide/services/sssd/service_sssd_enabled/rule.yml

```diff
@@ -17,8 +17,6 @@ identifiers:
     cce@rhel7: 80363-5
     cce@rhel8: 82440-9
 
-platform: machine
```

Comment:
Containers have services too. Reverting this change.

Reply:
ditto (see `chronyd_or_chronyc_network`)